sdk/python/README.md

```diff
@@ -9,12 +9,12 @@ Python 2.7 and 3.5+
 ### pip install
 
 ```sh
-pip install kubeflow-tfjob
+pip install kubeflow-training
```

Comment:
Do you publish this lib?

Reply:
No, I haven't published it. I just updated the readme with the new package name.